Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -231,16 +244,12 @@
 
 - type: entity
   id: AutolatheHyperConvection
-  parent: Autolathe
+  parent: [Autolathe, BaseHyperlathe]
```

Comment:
does it not inherit the component definition from `Autolathe` first, before getting the one from `BaseHyperLathe`

Reply:
i think its the opposite??? will need to test

Reply:
test soon i forgor

Reply:
no im right BaseHyperlathe comes after, the resulting prototype has 0.5 usage 1.5 time as expected